src/routes/PublicLayoutRouter/PublicLayoutRouter.js

```diff
@@ -28,9 +28,8 @@ export default function PublicLayoutRouter (props) {
         <Route path={`/${POST_DETAIL_MATCH}`} exact component={PublicPostDetail} />
         <Route path='/:context(groups)/:groupSlug' exact component={PublicGroupDetail} />
         <Route path='/:context(public)/:view(map)' component={MapExplorerLayoutRouter} />
-        <Route path='/:context(public)/:view(groups)' exact component={GroupExplorerLayoutRouter} />
-        {/* Remove this once we show the public stream */}
-        <Redirect exact from={`/public/${POST_DETAIL_MATCH}`} to='/post/:postId' />
+        <Route path='/:context(public)/:view(groups)' component={GroupExplorerLayoutRouter} />
+        <Redirect from={`(.*)/${POST_DETAIL_MATCH}`} to='/post/:postId' />
```

Comment:
won't this break trying to look at a post from the public map? it will redirect to /post/id

Reply:
any public/map/post/:postId routes will be caught and forwarded to the 'MapExplorerLayoutRouter' before this redirect is invoked. AND... the mapExplorer is wonky after the routing shifts, which I am now addressing in the context of the map not rendering.

Reply:
ok cool